lib/app/utils/assertions.rb

```diff
@@ -101,9 +101,9 @@ def assert_resource_content_type(client_reply, content_type)
       response_content_type = header
       response_content_type = header[0, header.index(';')] unless header.index(';').nil?
 
-      return if assertion_negated(response_content_type == "application/fhir+#{content_type}")
```

Comment:
If we are fixing this, also rename the assertion to `assert_response_content_type_equals`.  Having `resource` in the method name makes no sense, and since it is checking the content_type is something specific that is passed in, we need to be more specific. We have to be careful about changing the behavior of shared assertions, but it doesn't appear that this one is used anywhere else.